webapp/src/components/sidebar_right/sidebar_right.tsx

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import Scrollbars from 'react-custom-scrollbars';
+import {Theme} from 'mattermost-redux/types/preferences';
+import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+
+import {RHSStates} from '../../constants';
+import GitlabItems from '../sidebar_right/gitlab_items';
+import {Item} from '../../types/gitlab_items';
+
+interface PropTypes{
+    username: string;
+    org: string;
+    gitlabURL: string;
+    reviews: Item[];
+    unreads: Item[],
+    yourPrs: Item[],
+    yourAssignments: Item[],
+    rhsState: string,
+    theme: Theme,
+    actions:any,
+};
+
+export function renderView(props:PropTypes) {
```

Comment:
We add a space before defining the type. Change this everywhere.
```suggestion
export function renderView(props: PropTypes) {
```